data-collection/src/main/java/refactoringml/RunSingleProject.java

```diff
@@ -46,6 +48,7 @@ public static void main(String[] args) throws Exception {
 			gitUrl = args[1].trim();
 			storagePath = lastSlashDir(args[2].trim());
 
+			//TODO: is this extension necessary, it is inconsistent with the url handling in RunQueue
```

Comment:
It was, at least in my machine and in TU Delft servers. And I solved it via an ugly workaround, as you can see!

We can make the user to pass the extra db parameters, if required in their environment, i.e., adding a new optional args for these parameters.